apps/transport/lib/transport/history.ex

```diff
@@ -0,0 +1,95 @@
+defmodule Transport.History do
+  @moduledoc """
+  backup all ressources into s3 to have an history
+  """
+  alias ExAws.S3
+  alias Transport.{Repo, Resource}
+  import Ecto.{Query}
+  require Logger
+
+  def backup_resources() do
+    if Application.get_env(:ex_aws, :access_key_id) == nil
+      || Application.get_env(:ex_aws, :secret_access_key) == nil do
+      # if the cellar credential are missing, we skip the whole history backup
+        Logger.warn("no cellar credential set, we skip to resource backup")
+    else
+      resources_to_backup = Resource
+      |> where([r], not is_nil(r.url) and not is_nil(r.title) and (ilike(r.format, "GTFS") or ilike(r.format, "netex")))
```

Comment:
Check by equality, not like (btw, without `%` it’s an equality)

Reply:
but that's a *i*like here, so that's different no ?

but yes I think we can strictly compare